### ISSUE TITLE
Labels format

### ISSUE THREAD

**Issue by aurrelhebert** [senx/warp10-platform#629]
Hello, 

    With the master Ingress version, labels urlencoded including a `+` as 

```
// spaceplus2B{version=2.3.0%2B+store} 42
```

   are pushed with 2 `+` characters inside instead of a plus and a space in previous version

Query result:
```json
 {
      "c": "spaceplus2B",
      "l": {
        ".app": "ovh.unicorn",
        "version": "2.3.0++store"
      },
      "a": {},
      "la": 1578992305371,
      "v": [
        [
          1578992306410443,
          42
        ]
      ]
    }
```

**Comment by hbs** [comment]
This was actually a bug, strings in Warp 10 are expected to be percent encoded, not URL encoded, so 'x+y' is itself and not 'x y', the fact that 'x%2By+z' was converted to 'x+y z' was a bug present since many versions.

To mitigate the impact of the change, we can introduce a configuration which will force the previous buggy behavior, i.e. URL decode (thus transforming the + into a whitespace) the strings in parseLabelSelectors when '%' is present in the string.

But it is strongly advised that you encode spaces as '%20' as they should be in Warp 10 as only percent encoding is handled.

**Comment by aurrelhebert** [comment]
Hi thanks for the quick answer :). 

 That's what I thought too at first sight. I patched our tooling to push spaces as "%20". Afterwards I discover that some native library (as Go one for example) are URL encoding spaces as `+`.

**Comment by hbs** [comment]
pure URL encoding uses + for whitespace, but Warp 10 supports percent encoding, not URL encoding which leads to awkward syntaxes when specifying regexps for example.

It is better to generate %20 for whitespace, nevertheless we will provide this workaround to restore the buggy behavior.

**Comment by hbs** [comment]
I think @ftence comment on the PR makes perfect sense, the initial modification introduced by #620 was to fix a long lasting bug which could cause incorrect GTS to be deleted due to incorrect decoding of '+' characters. The behavior made possible by #630 would actually reactivate the ambiguity corrected by #620, therefore we will not apply #630. It is nevertheless available on GitHub should you want to re-introduce the pre #620 behavior.

**Comment by aurrelhebert** [comment]
I do agree with @ftence and the reason that lead to this fix! It will be great to have also a post or something a user can refers to avoid any impact of this breaking change.
